tests/test_examples/test_runs_via_configs/configs/train_postprocessor.yaml

```diff
@@ -13,6 +13,10 @@ num_workers: 0
 max_epochs: 2
 valid_period: 1
 
+# EXTRACTION
+
+embeddings_cache_dir: /tmp
```

Comment:
Do we need not a `null` value of `embeddings_cache_dir` for tests?

Reply:
this thing here is to check that saving is working (not to for optimization)